.pre-commit-hooks.yaml

```diff
@@ -1,11 +1,11 @@
 - id: check-message
   name: check commit message
   description: requiring commit message to match regex
-  entry: env IS_PRE_COMMIT=1 commit-check
+  entry: commit-check
   args: [--message]
   pass_filenames: false
```

Comment:
My idea was to let pre-commit pass the path to the log file containing the commit message. To do this, I think `pass_filename` should be set to `true`.

More info at
- [git docs for `commit-msg` hook](https://git-scm.com/docs/githooks#_commit_msg)
- [git docs for `prepare-commit-msg`](https://git-scm.com/docs/githooks#_prepare_commit_msg)

I'm not sure how the positional arguments from the underlying git hooks could be used by `commit-check` if `pass_filenames` is `false`. Although, I'm not very familiar with how pre-commit works under the hood.

Reply:
See also [pre-commit docs about how arguments are passed](https://pre-commit.com/#arguments-pattern-in-hooks) to the check-commits exe/script.